tests/models/test_tpu.py

```diff
@@ -383,3 +383,18 @@ def test_tpu_precision_16_clip_gradients(mock_clip_grad_norm, clip_val, tmpdir):
         mock_clip_grad_norm.assert_called()
     else:
         mock_clip_grad_norm.assert_not_called()
+
+
+@RunIf(tpu=True)
+@pl_multi_process_test
```

Comment:
noob question: what does this do?

Reply:
TPU attach the tpu_cores to the process. pl_multi_process_test Popen the function into a subprocess, so pytest doesn't hang as tpu_cores would have already been assigned.